src/pages/[...slug].tsx

```diff
@@ -182,7 +187,12 @@ const ContentPage: NextPage<ContentPiece> = ({
     );
 
     const contentBody = (
-        <>
+        <div
+            sx={{
+                ...middleSectionStyles,
+                my: ['section20', null, 'section30', 'section40'],
```

Comment:
what is my ?

Reply:
It's an 'sx' shorthand for 'marginTop' and 'marginBottom'. Like margin on the y axis.

Reply:
Nice :). you are SX champ.